nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -149,9 +149,18 @@
    </listitem>
    <listitem>
     <para>
-     Several of the apache subservices have been replaced with full NixOS
-     modules including LimeSurvey, WordPress, and Zabbix.
-     These modules can be enabled using the <option>services.limesurvey.enable</option>,
+     The apache service no longer attempts to start the postgresql service.
```

Comment:
s/apache/httpd/

+ a comment about how to make it depend upon postgresql again?